app/models/cart.rb

```diff
@@ -9,7 +9,39 @@ def add_item(item_id)
     contents[item_id.to_s] += 1
   end
 
+  def sum
+    drinks.map {|total|total.sum}.reduce(:+)
```

Comment:
Space after closing `|` missing.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:sum` as an argument to `map` instead of a block.